pilot/pkg/model/authentication.go

```diff
@@ -47,6 +51,31 @@ func GetConsolidateAuthenticationPolicy(mesh *meshconfig.MeshConfig, store Istio
 	return legacyAuthenticationPolicyToPolicy(mesh.AuthPolicy)
 }
 
+// ConstructSdsSecretConfig constructs SDS Sececret Configuration.
+func ConstructSdsSecretConfig(serviceAccount string, meshConfig *meshconfig.MeshConfig) *auth.SdsSecretConfig {
```

Comment:
I would suggest to change the function to take sdsRefreshDelay and sdsUdsPath instead of meshConfig. It's clearer what is used from the meshConfig, and make it easier for testing.

Reply:
updated.